DDDEastAnglia/Views/Home/Index.cshtml

```diff
@@ -35,6 +35,7 @@
                     <li><a href="http://www.slideshare.net/boyney123/web-developers-are-now-mobile-developers">Web Developers Are Now Mobile App Developers (David Boyne)</a></li>
                     <li><a href="http://www.slideshare.net/Nathangl/you-keep-using-the-word-agile-i-do-not-think-it-means-what-you-think-it-means">You keep using the word agile, I do not think it means what you think it means (Nathan Gloyn)</a></li>
                     <li><a href="http://colinmackay.scot/2015/09/26/continuous-delivery-ddd-east-anglia/">Continuous Delivery (Colin Angus Mackay)</a></li>
+                    <li><a href="https://onedrive.live.com/redir.aspx?cid=4f49eba58e1d0c02&resid=4F49EBA58E1D0C02!352352&parId=4F49EBA58E1D0C02!127&authkey=!AGMBKbMOFsSwETg&ithint=file%2cpptx">How my life changed when I met Git (it wasn't love at first sight) (Matteo Emili)</a></li>
```

Comment:
Could you move it up to after David Boyne's session, as I've the put the existing ones in order as they are in the agenda.